rewrite-java-core/src/main/java/com/azure/recipes/core/v2/TypeReferenceRecipe.java

```diff
@@ -119,14 +119,14 @@ private static class ConvertTypeReferenceVisitor extends JavaIsoVisitor<Executio
             boolean isNewImport = importSet.add(importQualid);
 
             // If the import is for ParameterizedType and it's already in the set, skip it
-            if (importQualid.equals("java.lang.reflect.ParameterizedType") && !isNewImport) {
+            if (importQualid.equals(" java.lang.reflect.ParameterizedType") && !isNewImport) {
```

Comment:
do we need the space after "?

Reply:
Space is needed to ensure import is updated correctly. Without the space, the import would be updated like so:

`importjava.lang.reflect.ParameterizedType;`

With the space, the import is updated like so:

`import java.lang.reflect.ParameterizedType;`

Reply:
I think this is just checking, so we could use importQualid.trim() and check for equality.